app/controllers/assignments_controller.rb

```diff
@@ -1,5 +1,6 @@
 class AssignmentsController < ApplicationController
-  before_action :set_assignment, only: [:show, :edit, :update, :destroy]
+  before_action :set_assignment, only: [:show, :edit, :update, :destroy,
+                                        :last_submitted_hours_and_feedbacks]
   before_action :set_volunteer, only: [:find_client]
```

Comment:
I would say it is more explicit to call `:set_volunteer` within `find_client`, since you're only calling it for that action ... same for the `before_action :set_client, only: [:find_volunteer]`